data/Music/Sonic the Hedgehog Series/Sonic the Hedgehog 1/final.meta

```diff
@@ -1,3 +1,2 @@
-Sonic the Hedgehog - Final Zone
-0x03
-bgm
```

Comment:
Were the changes to this file intended?

Reply:
I either forgot to update it, or didn't have a good location at the time and forgot to remove it. I have fixed this